packages/tinybased/src/lib/tinybased.ts

```diff
@@ -90,6 +92,37 @@ export class TinyBased<
     }
   }
 
+  public onRowAddedOrUpdated(
+    handler: RowChangeHandler<TBSchema>,
+    canMutate = false
+  ) {
+    this.store.addRowListener(
+      null,
+      null,
+      (store, table, rowId) => {
+        if (store.hasRow(table, rowId)) {
+          const row = store.getRow(table, rowId);
+
+          handler(table, rowId, row as any);
+        }
+      },
+      canMutate
+    );
```

Comment:
You can do this
```suggestion
    this.events.onRowAddedOrUpdated.add(handler);
```
so the same logic on lines 82-91 can be the reference, you can expand the Set so it can take additional options

Reply:
Maybe something like this 
```
this.events.onRowAddedOrUpdated.add({ handler, canMutate });
```

Reply:
It's not as simple as that, unfortunately.

These row listeners being added must have support to mutate, which is the last optional parameter to `addRowListener`. They're also going to be added _after_ the tinybase instance has been initialized.